src/data/mastering-bitcoin-cash/3-keys-addresses-wallets.md

```diff
@@ -84,25 +84,19 @@ Elliptic curve cryptography is a type of asymmetric or public-key cryptography b
 
 Bitcoin Cash uses a specific elliptic curve and set of mathematical constants, as defined in a standard called secp256k1, established by the National Institute of Standards and Technology (NIST). The secp256k1 curve is defined by the following function, which produces an elliptic curve:
 
-```
-\[\begin{equation} {y^2 = (x^3 + 7)}~\text{over}~(\mathbb{F}_p) \end{equation}\]
-```
+![secp256k1 curve function](<https://latex.codecogs.com/svg.latex?\large&space;{y^2&space;=&space;(x^3&space;+&space;7)}~\text{over}~(\mathbb{F}_p)>)
 
 or
 
-```
-\[\begin{equation} {y^2 \mod p = (x^3 + 7) \mod p} \end{equation}\]
-```
+![alternate secp256k1 curve function](<https://latex.codecogs.com/svg.latex?\large&space;{y^2&space;\mod&space;p&space;=&space;(x^3&space;+&space;7)&space;\mod&space;p}>)
 
 <spacer size="small"></spacer>
 
-```
-The mod p (modulo prime number p) indicates that this curve is over a finite field of prime order _p_, also written as \(\(\mathbb{F}_p\)\), where p = 2256 – 232 – 29 – 28 – 27            – 26 – 24 – 1, a very large prime number.
-```
+> The `mod p` (modulo prime number p) indicates that this curve is over a finite field of prime order `p`, also written as &#120125;<sub>p</sub>, where `p` = 2<sup>256</sup> – 2<sup>32</sup> – 2<sup>9</sup> – 2<sup>8</sup> – 2<sup>7</sup> – 2<sup>6</sup> – 2<sup>4</sup> – 1, a very large prime number.
```

Comment:
This line in particular needed an update.  Without `<sup>` tags or `^` symbols, there's no indication that these numbers are exponents.